woof-distro/arm/debian/bullseye/packages-templates/firmware-brcm80211/lib/firmware/brcm/brcmfmac4354-sdio.txt

```diff
@@ -0,0 +1,155 @@
+# Sample variables file for BCM94354Z NGFF 22x30mm iPA, iLNA board with PCIe/SDIO for production package

```

Comment:
TODO: see if I can download this file from the Chrome OS kernel repo during the build, instead of holding a copy